docker/release-ci/Dockerfile_sample

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:XX.04
```

Comment:
These template variables should avoid name collisions: perhaps CYCLUS_UBUNTU_MAJOR_VERSION, CYCLUS_REPO_HASH